compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -297,6 +301,78 @@ static SmallVector<int64_t> setInnerPermB(bool isMatmulTransposeB) {
   return innerPermB;
 }
 
+/// TODO(avarma): This currently is skipping checking for ext* ops.
+static bool bodyMatcherForMatmulTranspose(Value yieldVal, Block *body) {
```

Comment:
Did this function need to move upwards in the .cpp? It's just that it makes reviewing a bit trickier, as I'm not sure if this function has changed.

Reply:
Thanks. I have no strong preference so if you think there's a good reason to move it, then go ahead.